auditlog/registry.py

```diff
@@ -266,7 +266,8 @@ def register_from_settings(self):
         """
         if not isinstance(settings.AUDITLOG_INCLUDE_ALL_MODELS, bool):
             raise TypeError("Setting 'AUDITLOG_INCLUDE_ALL_MODELS' must be a boolean")
-
+        if not isinstance(settings.AUDITLOG_DISABLE_ON_RAW_SAVE, bool):
+            raise TypeError("Setting 'AUDITLOG_DISABLE_ON_RAW_SAVE' must be a boolean")
```

Comment:
We need to have a test for this line. something like https://github.com/jazzband/django-auditlog/blob/aa6d977f8b8b070690f9a168f2eddaa93a65df70/auditlog_tests/tests.py#L1043-L1047

Reply:
Good point, I've fixed that